TheMovieDb/Home/Contoller/MainTabViewController.swift

```diff
@@ -7,8 +7,8 @@
 
 import UIKit
 
+let categoryHomeHeaderId = "categoryHomeHeaderId"
```

Comment:
this is a global object, instead save the constant inside the class you need it or make a struct constants

Reply:
moved to internal class